packages/export-cli/src/gulp/serve.js

```diff
@@ -9,13 +9,21 @@ const lint = require('./lint');
 const server = require('./server');
 
 module.exports = (cwd, serverFile) => () => {
+  if (process.env.GULP_POLLING_ENABLED) log('Falling back to polling to watch files!');
   const watcher = watch(
     ['**/*.js'],
     {
       cwd,
       queue: false,
       ignoreInitial: false,
       ignored: ['node_modules'],
+      ...(process.env.GULP_POLLING_ENABLED && {
+        // Use polling on windows https://forums.docker.com/t/file-system-watch-does-not-work-with-mounted-volumes/12038/16
```

Comment:
It looks like polling is going to cause significant resource issues. Have you explored the option presented in [this](https://forums.docker.com/t/file-system-watch-does-not-work-with-mounted-volumes/12038/21) comment (from the same thread)?

Reply:
I didn't because I didn't want to introduce the extra dependencies it would require: a working Python environment, the `pip` package manager, and the `docker-windows-volume-watcher` utility itself.